ci/xtask/Cargo.toml

```diff
@@ -1,6 +1,10 @@
 [package]
 name = "xtask"
+<<<<<<< HEAD
 version = "3.1.0-beta.0"
+=======
+version = "0.1.0"
+>>>>>>> fdecf73aa (Update xtask version to 0.1.0 since we will never publish the xtask crate (#8950))
```

Comment:
Please try to resolve merge conflicts before requesting reviews 😄

Reply:
Sorry about that. Resolved